flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ddl/AlterTablePropertiesOperation.java

```diff
@@ -26,27 +27,28 @@
 
 
 /**
- * Operation to describe a ALTER TABLE .. SET .. statement.
+ * Operation to describe a ALTER TABLE/VIEW .. SET .. statement.
  */
 public class AlterTablePropertiesOperation extends AlterTableOperation {
```

Comment:
Create a dedicated `AlterViewPropertiesOperation` and extends from `AlterViewOperation`